### ISSUE TITLE
paste from cURL should not require -X arg

### ISSUE THREAD

**Issue by anentropic** [a85/POSTMan-Chrome-Extension#3002]
1. Postman Version: 4.10.7
 2. App (Chrome app or Mac app): Mac app
 3. OS details: macOS Sierra

I use Postman in conjunction with the Charles proxy app in my day to day development work.

Charles provides an option to copy a request as a cURL command.  And Postman provides an option to import a request by pasting the text of a cURL command.

Unfortunately this doesn't work for GET requests.  It seems that, for GET requests, Charles omits the `-X GET` parameter and Postman complains the command is invalid.  I believe that GET is the default value and the command _is_ valid.  Manually adding a `-X GET`  to the pasted command allows Postman to import it.  Other kinds of requests work fine as Charles includes the `-X PATCH` or whatever.

So it'd be nice if Postman could recognise cURL commands that don't have an explicit `-X GET` arg as being GET requests, allowing seamless pasting between Charles and Postman.

**Comment by abhijitkane** [comment]
@anentropic `curl http://www.getpostman.com` works in Postman's import window. Can you share an example of a curl command that you're unable to import?

**Comment by anentropic** [comment]
Here is one:

```
curl -H 'User-Agent: Depop Android X QA 1.35 rv:17089560 (E5823 - 7.0; * en_GB)' -H 'X-Garage-bundle-id: com.depop.qa' -H 'X-Country-Code: GB' -H 'Accept-Language: en-GB' -H 'Content-Type: application/json' -H 'Host: api-staging.depop.com' -H 'If-Modified-Since: Mon, 08 May 2017 09:12:26 GMT' --compressed 'https://api-staging.depop.com/api/v1/seller-info'
```

Hmm... above I said that adding `-X GET` allowed import (with a different example)

But I can't reproduce that with this one, still gives error

Instead, removing the `--compressed` arg allows Postman to import it

**Comment by chappellnick** [comment]
I have this same issue with Charles Proxy. Postman does not like the format which it outputs its curl requests.

**Comment by g4r37h** [comment]
Nearly a year later, this is still an issue.

I don't have time to manually modify cURL requests until Postman is willing to import them so I looked for an alternative. I found Paw. That worked with Charles right out of the gate.

**Comment by abhijitkane** [comment]
@anentropic @chappellnick @g4r37h Apologies for the delay on this. This has been fixed in the latest version of Postman - 7.2.1.
